### ISSUE TITLE
[Bug] Some post params are not imported from curl

### ISSUE THREAD

**Issue by eMerzh** [getinsomnia/insomnia#2057]
When importing request from curl, it seems that we drop some --data params
 
**To Reproduce**

- Create a new request 
- select POST, Form URL Encoded
- add param key "Hello" value "World"
- add param key "Foo" value "Baz"
- click copy as curl

- then create a new request
- past the curl in the url to import it

Only Hello=World is reported in query params

query : 

```bash
curl --request POST \
  --url http://example.com/ \
  --header 'content-type: application/x-www-form-urlencoded' \
  --data Hello=world \
  --data foo=baz
```

after import

<img width="644" alt="image" src="https://user-images.githubusercontent.com/177003/80388529-4c0d9d80-88aa-11ea-8523-562def3c3f20.png">

**Expected behavior**
Hello=World
and Foo=Baz  are imported

**Desktop (please complete the following information):**
 - OS: 10.15.3 (19D76)
 - Installation Method: dmg
 - App Version:  7.1.1 (7.1.1.3350)

**Comment by reynolek** [comment]
Thanks for opening a bug report! Can you paste an exact curl request that shows the behavior you are seeing? It will make it easier to be reproduced.

**Comment by eMerzh** [comment]
of course, i've updated the content

**Comment by reynolek** [comment]
Looks like the bug is here: https://github.com/Kong/insomnia/blob/develop/packages/insomnia-importers/src/importers/curl.js#L248-L255

The function is returning the first match (in the example `hello=world`) and ignores the fact that there is another `--data`.

That being said, I think the other issue is that [this piece of code](https://github.com/Kong/insomnia/blob/develop/packages/insomnia-importers/src/importers/curl.js#L214-L218) sees `--header 'content-type: application/x-www-form-urlencoded'`, and then processes the data as if they were `&` together. I confirmed that insomnia works as expected if you use `--data "hello=world&foo=baz"`.

**Comment by eMerzh** [comment]
@reynolek yess i've create a PR for this in #2058

**Comment by reynolek** [comment]
Ah I hadn't refreshed this issue until this comment happened. Ill tag the devs for review.

**Comment by develohpanda** [comment]
Resolved with #2058 and #2070 🙂
